extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampRadioStreamExtractor.java

```diff
@@ -78,11 +80,9 @@ public String getUrl() throws ParsingException {
 
     @Nonnull
     @Override
-    public String getUploaderName() throws ParsingException {
-        return Jsoup.parse(showInfo.getString("image_caption")).getElementsByTag("a").stream()
-                .map(Element::text)
-                .findFirst()
-                .orElseThrow(() -> new ParsingException("Could not get uploader name"));
+    public String getUploaderName() {
+        return Jsoup.parse(showInfo.getString("image_caption"))
+                .getElementsByTag("a").first().text();
```

Comment:
Why did you revert back to not using Java 8 streams? I think this is a rebasing error